### DISCUSSION TITLE
Parameters of Get-Clipboard

### DISCUSSION THREAD

**Discussion by kvprasoon** [PowerShell/PowerShell#15170] [Q&A]
Just noticed that `Get-Clipboard` has only `-Raw` parameter in 7.x. Was other parameters like `-Format` and `-TextFormatType` intentionally removed ?

**Comment by ninmonkey** [comment]
It looks like `Get-ClipBoard` doesn't exist in PS6? <https://docs.microsoft.com/en-us/previous-versions/powershell/module/microsoft.powershell.management/get-clipboard?view=powershell-6.0>

If true, then it might make sense because PS5 uses `dotnet` where Powershell uses `dotnet core` -- and must work cross-platform, So not all apis are the same.

**Reply by vexx32** [reply]
Yeah. The `Get-Clipboard` cmdlet in Windows PowerShell uses some windows-only APIs, which is partially why it's able to provide more specific modes. The cross-platform .NET APIs for this, as far as I'm aware, only really support copy/pasting text.

It was necessarily removed for 6.0, I guess the team hadn't had time to reimplement it. By 6.1 or thereabouts the current cmdlet was written to use the cross-platform APIs, but the Windows-specific functionality was never re-added.
